telegram/_poll.py

```diff
@@ -104,22 +115,24 @@ class PollAnswer(TelegramObject):
 
     """
 
-    __slots__ = ("option_ids", "user", "poll_id")
+    __slots__ = ("option_ids", "user", "poll_id", "voter_chat")
 
     def __init__(
         self,
         poll_id: str,
+        voter_chat: Chat,
         user: User,
         option_ids: Sequence[int],
```

Comment:
The order of these two has to change as `user` is now optional. We can provide a backward compatibility layer by checking the type of the arguments `user` and `option_ids`.

Reply:
do we also issue a warning in such cases?